src/allmydata/web/info.py

```diff
@@ -36,6 +36,8 @@ def get_type(self):
                     return ret
                 return "immutable file"
             return "immutable LIT file"
+        if IVerifyNode.providedBy(node):
+            return "verifier"
         return "unknown"
```

Comment:
This will have the same problem of never returning "unknown" because ``CiphertextFileNode`` implements ``IVerifyNode``.